webapps/src/main/java/org/camunda/bpm/webapp/impl/security/filter/CsrfPreventionFilter.java

```diff
@@ -101,11 +102,9 @@
 
   private int denyStatus = HttpServletResponse.SC_FORBIDDEN;
 
-  private final Set<String> entryPoints = new HashSet<String>();
+  private final Set<String> entryPoints = new HashSet<>();
```

Comment:
```suggestion
  protected final Set<String> entryPoints = new HashSet<>();
```
Should we also change visibility when we touch this line anyways?